process-document/src/main/kotlin/com/ritense/processdocument/service/ProcessDocumentKotlinService.kt

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2015-2023 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ritense.processdocument.service
+
+import com.ritense.document.domain.Document
+import com.ritense.document.domain.impl.JsonSchemaDocumentId
+import com.ritense.document.exception.DocumentNotFoundException
+import com.ritense.document.service.DocumentService
+import com.ritense.valtimo.service.CamundaProcessService
+import java.util.*
+
+class ProcessDocumentKotlinService(
```

Comment:
I don't think we should include Kotlin in the name of this service.

Reply:
there is already an interface called ProcessDocumentService.. i couldn't think of other names. i'm open to suggestions

Reply:
i guess this is the best i can do